examples/solvers/douglas_rachford_pd_tomography_tv.py

```diff
@@ -84,7 +83,7 @@
 # Functional to enforce 0 <= x <= 1
 f = odl.solvers.IndicatorBox(space, 0, 1)
 
-# Functional for I_{Ax = g}
+# Functional to enforce Ax = g
```

Comment:
Now that I look at it again, it may also confusing for a user to see `Ax = g` here but no `A` in the functional. So it would be good to add a comment that due to splitting, the operator A is not included here.